classes/MyTableLister.php

```diff
@@ -229,7 +229,7 @@ public function selectSQL($columns, &$vars)
                 $foreign_link = isset($tmp[$foreign_link]) && $foreign_link ? $foreign_link : reset($tmp);
                 $columns[$key] = $this->escapeDbIdentifier($this->table) . '.' . $value . ','
                         . $this->escapeDbIdentifier($this->fields[$key]['foreign_table']) . '.'
-                        . $this->escapeDbIdentifier($foreign_link) . ' AS ' . $this->escapeDbIdentifier($key . $this->DEFAULTS['FOREIGNLINK']);
+                        . $this->escapeDbIdentifier($foreign_link) . ' AS `' . $this->escapeDbIdentifier($key . $this->DEFAULTS['FOREIGNLINK']) . '`';
```

Comment:
Tady se &#96;...&#96; mohou potenciálně zdvojit.
Radši jsem se na to koukl do zdrojáků a Tools::escapeDbIdentifier() obaluje identifikátor &#96;...&#96; vždy, LogMysqli::escapeDbIdentifier() ho obaluje &#96;...&#96; jen, když identifikátor obsahuje něco jiného než písmeno, číslici a podtržítko. (Což mi ponouká, že bych to měl v Tools:: udělat stejně jako je teď v LogMysqli::)
A správný zápis mi z toho vychází tedy jako zavolání prostého LogMysqli::escapeDbIdentifier() - bez &#96;...&#96;.
Mohl bys to, moc prosím, upravit (předpokládám, že to budeš mít rychleji)? Možná i zkouknout výskyty LogMysqli::escapeDbIdentifier() v souboru a zkontrolovat, že se okolo nění &#96;...&#96;.
Díky moc.

Reply:
udělal jsem commit, který `` odstranil